### PR TITLE
HCL BigFix with latest updates

### PR DESCRIPTION
**Pull request by john-talbert**
Removed duplicate Compliance and Inventory records 
Updated all BigFix module entries with note about the related CVE-2021-45046 as not vuln.
Fixed Alpha sort issue with Helpsystems lines.

Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - Status: please select a value from the status table at the top
  - Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - Please mind the sorting
  - Please put vendor/product name in PR title (instead of "Update README.md")

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -890,21 +890,17 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
...
-| HCL Software | BigFix Compliance | Unknown | Workaround | | [source](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0095486)|
-| HCL Software | BigFix Inventory | Unknown | Workaround | | [source](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0095486)|
+| HCL Software | BigFix Insights | Unknown | Not vuln | also not vuln for CVE-2021-45046 | [source](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0095486)|
+| HCL Software | BigFix Insights for Vulnerability Remediation | Unknown | Not vuln | also not vuln for CVE-2021-45046 | [source](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0095486)|
+| HCL Software | BigFix Compliance | > 2.0.1 ; < 2.0.4 | Workaround | also not vuln for CVE-2021-45046 | [source](https://forum.bigfix.com/t/bigfix-compliance-has-a-remediation-for-log4j-vulnerability-cve-2021-44228/40197)|
+| HCL Software | BigFix Inventory | < 10.0.7 | Workaround | also not vuln for CVE-2021-45046 | [source](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0095486)|
```
Comment (maertsen) [inline software/README.md:896]: if there's a workaround, I think should remove 'also'.

**Review by maertsen — CHANGES_REQUESTED** [review]
thanks @john-talbert, could you have a look at one small nit?

File: software/README.md (hunk 1 above)
Comment (john-talbert) [reply software/README.md:896]: Thank you.  
There is a workaround for CVE-2021-44228
Not Affected by CVE-2021-45046
I agree with your suggestion to remove also and will adjust

**Review by john-talbert — COMMENTED** [review]
Updated this pull to remove the word also

**Comment by john-talbert** [comment]
removed Heimdal line from my merge.

**Pull request merged**